### PR TITLE
LSR-986 Wrong UserPortraitScreenlet avatar in long lists

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/comment/display/CommentDisplayView.java [hunk 1]
```diff
@@ -87,6 +87,7 @@ private void loadView(boolean editable) {
 			enableEditionButtons(editable);
 			showEditionButtons();
 			userPortraitScreenlet.setUserId(commentEntry.getUserId());
+			userPortraitScreenlet.load();
 			userNameTextView.setText(commentEntry.getUserName());
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/comment/display/CommentDisplayView.java]: Maybe we can check here if the current userId and the userportrait's current userId are the same and not load the portrait? what do you think?

**Comment by victorg1991** [comment]
LGTM 👍  I've added a comment :)

**Comment by dgarciasarai** [comment]
It's not a bad idea. I will try it. Thanks @victorg1991 ;)

**Comment by dgarciasarai** [comment]
@victorg1991 I've just tried it and it's not very effective...If you check if userId it's not the same as the user portrait's current userId, load the "new" avatar with autoload set to false, the firsts rows are empty (default user portrait). And if the autoload is set to true, we have the initial problem again.

**Comment by nhpatt** [comment]
This is still missing some commits, right?

**Pull request merged**